.gitignore

```diff
@@ -10,3 +10,4 @@
 *.log
 .idea
 .vscode
+package-lock.json
```

Comment:
It doesn't need to be ignored as it wouldn't be generated because of `package-lock=false`

Reply:
But if it is not ignored, wouldn't existing projects push package-lock.json (which was previously created before this PR) again?

Reply:
@nveenjain Yes, I was thinking of that too...